### ISSUE TITLE
WCAG compliance

### ISSUE THREAD

**Issue by custa1200** [ment-mx/Prism#9]
Wonder if there is a way to nicely show the level of contrast re WCAG 2.

Could be:
- color on white background
- color on black background
- white on color background
- black on color background
- small size
- large size

**Comment by Adrxx** [comment]
Sorry, could you explain a little more what WCAG 2 is?
